tools/gradle-plugin/src/main/java/io/smallrye/openapi/gradleplugin/Configs.java

```diff
@@ -240,6 +244,11 @@ public OperationIdStrategy getOperationIdStrategy() {
                 return operationIdStrategy.getOrNull();
             }
 
+            @Override
+            public DuplicateOperationIdBehavior getDuplicateOperationIdBehavior() {
+                return duplicateOperationIdBehavior.getOrNull();
```

Comment:
Instead of null, can it default to the interface's default value? That way the logic checking this can always assume non-null.

Reply:
Done.